yt_dlp/extractor/playsuisse.py

```diff
@@ -5,10 +5,11 @@
 
 
 class PlaySuisseIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/watch/(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/(watch|detail)/(.*\?episodeId=)?(?P<id>[0-9]+)?'
```

Comment:
* Avoid `.*`
* Use non-capturing groups `(?:...)`
* There can be other query
* Why did you make `id` optional?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/(?:watch|detail)(?:[^#]*[?&]episodeId=)?(?P<id>[0-9]+)'
```

Reply:
Thanks for suggestion! I had to add a slash but otherwise works fine. Optional `id` was a mistake.